### DISCUSSION TITLE
OpenColorIO missing setup file

### DISCUSSION THREAD

**Discussion by llamafilm** [Homebrew/discussions#1072] [Everyday usage]
I installed OpenColorIO on Catalina with `brew install ocio` but the setup script is missing.  According to the [docs](https://formulae.brew.sh/formula/opencolorio):
> OpenColorIO requires several environment variables to be set.
You can source the following script in your shell-startup to do that:
    `$(brew --prefix)/share/ocio/setup_ocio.sh`

But this path doesn't exist.  All I have is this:

```
% brew list ocio
/usr/local/Cellar/opencolorio/2.0.0/bin/ociobakelut
/usr/local/Cellar/opencolorio/2.0.0/bin/ociocheck
/usr/local/Cellar/opencolorio/2.0.0/bin/ociochecklut
/usr/local/Cellar/opencolorio/2.0.0/bin/ociomakeclf
/usr/local/Cellar/opencolorio/2.0.0/bin/ociowrite
/usr/local/Cellar/opencolorio/2.0.0/include/OpenColorIO/ (7 files)
/usr/local/Cellar/opencolorio/2.0.0/lib/libOpenColorIO.2.0.0.dylib
/usr/local/Cellar/opencolorio/2.0.0/lib/pkgconfig/OpenColorIO.pc
/usr/local/Cellar/opencolorio/2.0.0/lib/python3.9/site-packages/PyOpenColorIO.so
/usr/local/Cellar/opencolorio/2.0.0/lib/ (3 other files)
```

**Comment by carlocab** [comment]
We just use OpenColorIO's build scripts to install it (i.e. `make` and `make install`), so this is missing because their `make install` doesn't install `setup_ocio.sh`. It might be better to report this upstream so their `make install` can be fixed.

**Reply by llamafilm** [reply]
Ok thanks, I reported it to them https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1341
This is my first time using OCIO, and it seems to require a lot of setup.

**Comment by llamafilm** [comment]
Do you know how to install the Python module? Maybe this is another question for upstream, but the installation doc says to run brew `install opencolorio --with-python`. This option doesn't work, and I don't see any args available in the Ruby formula.

**Reply by carlocab** [reply]
Yes, it seems their docs are very outdated. Options in homebrew/core were deprecated a couple of years ago. I believe OpenColorIO should install the Python module by default. If it doesn't, I'd be happy to review a PR that enables it. 

The formula isn't complicated, so if their documentation explains how to build ocio from source, it should'nt be hard to do this.
